webdriver/tests/bidi/browsing_context/traverse_history/invalid.py

```diff
@@ -42,3 +42,24 @@ async def test_delta_invalid_value(bidi_session, current_url, new_tab, inline, v
         await bidi_session.browsing_context.traverse_history(
             context=new_tab["context"], delta=value
         )
+
+
+@pytest.mark.parametrize("domain", ["", "alt"],
+                         ids=["same_origin", "cross_origin"])
```

Comment:
We no longer need the parameterization given that it doesn't matter what kind this iframe will be.

Reply:
done